src/gravity/gravity.cpp

```diff
@@ -31,22 +55,38 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   std::string sys = pin->GetOrAddString("artemis", "coordinates", "cartesian");
   Coordinates coords = geometry::CoordSelect(sys, ndim);
 
-  std::string grav_type = pin->GetString("gravity", "type");
+  // Find which gravity type is requested
+  // Note that we do not use if-elif statements to detect multiple gravity blocks
```

Comment:
Should we be? It was before, but I guess we might want point gravity and nbody enabled at the same time? Is that the thought process here?

Reply:
No, you can run into the situation of

```
<gravity/point>
<gravity/binary>
```
Then if your if-else is laid out as,
```
if (point) {
  reads point fine
} else if (binary) {
  never gets here and never tells the user to not define multiple nodes
}
```

The way I have it now, reads them all, and if there's more than one it fatals correctly.

Reply:
But also, yes, this allows the freedom in the future to have multiple types of gravity active.

Reply:
Oh now I see why... it's because you are invoking `DoesBlockExist("...")` in the clauses.